client/my-sites/comments/controller.js

```diff
@@ -29,30 +29,30 @@ const changePage = path => pageNumber => {
 	return page( addQueryArgs( { page: pageNumber }, path ) );
 };
 
-export const siteComments = ( { params, path, query, store } ) => {
-	const siteFragment = route.getSiteFragment( path );
+export const siteComments = context => {
+	const siteFragment = route.getSiteFragment( context.path );
```

Comment:
What about desctructuring around here (using `const { ... } = context;`), much like you're doing in `my-sites/sharing/controller` now? We're using `context.path` three times, would be nice if it was only `path` IMO.